nbsite/shared_conf.py

```diff
@@ -13,30 +13,53 @@ def setup(app):
         print('no param_formatter (no param?)')
 
     nbbuild.setup(app)
+    app.connect("builder-inited", remove_mystnb_static)
+
+def remove_mystnb_static(app):
+    # Ensure our myst_nb.css is loaded
+    app.config.html_static_path = [
+        p for p in app.config.html_static_path if 'myst_nb' not in p
```

Comment:
Comment might need a little clarification: how does setting `html_static_path` when 'myst_nb' is *not* in an element ensure the myst_nb.css is loaded?

Reply:
I'll clarify, we're ensuring that it does not load the `mystnb.css` from `myst_nb` but the custom version we are shipping.